src/settings/LocationLocations/LocationForm/LocationFormContainer.js

```diff
@@ -19,6 +26,34 @@ const LocationFormContainer = ({
     setInitialValues(location);
   }, [location?.id]);
 
+  const callout = useContext(CalloutContext);
+
+  function showSubmitErrorCallout(error) {
+    callout.sendCallout({
+      type: 'error',
+      message: error.message || error.statusText || <FormattedMessage id="ui-tenant-settings.settings.save.error.network" />,
+    });
+  }
+
+  const { setMapping } = useRemoteStorageApi();
+
+  const fireSetMapping = (...args) => setMapping(...args).catch(showSubmitErrorCallout);
```

Comment:
I suggest renaming it to something like `initiateSetMapping` for better readability.

Reply:
done